src/components/CardHeader.js

```diff
@@ -14,19 +14,13 @@ export default function CardHeader({ cardText, title, backClickHandler }) {
           style={{
             ...(backClickHandler
               ? { paddingLeft: colPaddingX, paddingRight: colPaddingX }
-              : {}),
+              : {})
           }}>
           {backClickHandler && (
             <ArrowLeft
+              className='back-arrow'
```

Comment:
We need to add this class for polkadot.scss as well if we are removing the style